client/src/components/ProTx/DashboardHeader.js

```diff
@@ -5,7 +5,8 @@ function DashboardHeader() {
   return (
     <div className="dashboard-header-root">
       <p>
-        <b>COOKS PRE-PRODUCTION PORTAL</b> Geospatial data analytics dashboard displaying incomplete demonstration data.
+        <b>COOKS PRE-PRODUCTION PORTAL</b> Geospatial data analytics dashboard
```

Comment:
Forcing line breaks in content text strings is a terrible idea. Can we adjust this specific check?